### DISCUSSION TITLE
change watermark color

### DISCUSSION THREAD

**Discussion by ghost** [ImageMagick/ImageMagick#4088] [Help]
- How to make the watermark color white? 
-   the watermark has transparent background and is black in color
-command used to place 'magick composite -gravity NorthWest -geometry +5+5  images/resized_nike.png  images/image'+str(i+1)+'.jpg images/wmark'+str(i+1)+'.jpg'

![wmark5](https://user-images.githubusercontent.com/20657303/129864451-7e80b11a-9fc1-4945-a072-c9941c32b8c1.jpg)

**Comment by snibgo** [comment]
What version of IM, on what platform?

I suggest you use "magick", not "magick composite", and use "-composite" after the inputs. The inputs should then be in the opposite order.

You do any processing to either input. Use parentheses to limit the processing to one input. For example, bash syntax:
```
magick \
  baseimage.jpg \
  \( wmark.png -fill White +opaque black \) \
  -gravity NorthWest -geometry +5+5 \
  -composite \
  out.jpg
```
If that doesn't work, please show your input files.

**Reply by ghost** [reply]
I want my output to look like as in the image below..i'm working on windows.
I got the output but is their any other way? 
- i processed the logo using -> magick convert  images/nike_black.png -channel RGB -negate -resize 50% images/resized_nike.png
- then used this to watermark ->  magick composite -compose Plus -gravity  NorthWest -geometry +5+5  images/resized_nike.png  images/image'+str(i+1)+'.jpg images/wmark'+str(i+1)+'.jpg

![wmark5](https://user-images.githubusercontent.com/20657303/129922418-6da89159-39c3-45c4-a0d6-0fda8dce98cb.jpg)

**Comment by snibgo** [comment]
You can insert the "-channel RGB -negate -resize 50%" operations into a single "magick" command, as I show above.

Please answer my questions:

> What version of IM, on what platform?

> ... please show your input files.
